src/test/java/uk/gov/justice/hmpps/prison/service/FinanceServiceTest.java

```diff
@@ -251,4 +261,126 @@ private TransferTransaction createTransferTransaction() {
                 .clientTransactionId("transId")
                 .build();
     }
+
+    @Test
+    public void test_getBalances_HappyPath() {
+
+        var bookingId = -1L;
```

Comment:
Minor thing but these should be **final** to ensure consistency with the rest of the file.

Reply:
done